.github/workflows/publish-specs.yml

```diff
@@ -79,4 +78,5 @@ jobs:
           export PATH="$(python3 -m site --user-base)/bin:${PATH}"
           bikeshed update
           mkdir out
-          ./compile.sh
+          cd wgsl
+          make
```

Comment:
Does this step ("Validate WGSL grammar") need to build the spec? If we only want the `grammar/grammar.js` target we should build that only. (Currently that doesn't depend on the spec building, but if it does in the future the Makefile will update to express that dependency and this will still be correct.)

Reply:
There is also the further important analyzer which is an independent target

Reply:
Ah, true. Maybe add a new PHONY target to the makefile called e.g. `validate` that runs the steps needed here?